labseu2-workout-tracker/server
# Ft start workout session

##  What does this PR do
This PR adds endpoint that records when a user starts a workout. Also adds a test for the endpoint

##  Description of task to be completed
- add controller for endpoint
- add route for endpoint
- update workout-session migration file to contain userId
- write tests for endpoint

##  How should this manually be tested
clone this repository
checkout  to branch ft-start-workout-session

Make sure to add a valid token to the Authorisation header

On Postman 
`http://localhost:5000/workouts/2/start `

To Test: `run npm test`

##  What are the relevant Trello board stories
[As a user, I want to be able to start a workout session](https://trello.com/c/G1elObQt/58-as-a-user-i-want-to-be-able-to-start-a-workout-session)
